docs/user-guide/01_getting_started/02_preprocess.md

```diff
@@ -11,12 +11,12 @@ PREFIDA([IDL](|url|/sourcefile/prefida.pro.html),[Python](|url|/sourcefile/prepr
 
 PREFIDA is called as follows
 ```idl
-IDL> prefida, inputs, grid, nbi, plasma, fields, dist, spec=spec, npa=npa
+IDL> prefida, inputs, grid, nbi, plasma, fields, fbm, spec=spec, npa=npa
```

Comment:
Change `fbm` back to dist. `fbm` has a specific meaning (a distribution function). `dist` is the more general term.